packages/ezflow_extension/ezextension/ezflow/modules/ezflow/zone.php

```diff
@@ -30,6 +30,9 @@
 $zoneID = $Params['ZoneID'];
 
 $contentObjectAttribute = eZContentObjectAttribute::fetch( $contentObjectAttributeID, $version );
+if ( !$contentObjectAttribute )
```

Comment:
Ah, I never know with legacy any more (all kinds of styles exist), but you're right, the correct form should use them, yes.